sci-libs/openblas/files/openblas-0.2.19-MAKE.patch

```diff
@@ -1,29 +0,0 @@
-diff --git a/Makefile b/Makefile
-index 2ae0047..2071e10 100644
---- a/Makefile
-+++ b/Makefile
-@@ -278,13 +278,13 @@ lapack-timing : large.tgz timing.tgz
- ifndef NOFORTRAN
- 	(cd $(NETLIB_LAPACK_DIR); $(TAR) zxf ../timing.tgz TIMING)
- 	(cd $(NETLIB_LAPACK_DIR)/TIMING; $(TAR) zxf ../../large.tgz )
--	make -C $(NETLIB_LAPACK_DIR)/TIMING
-+	$(MAKE) -C $(NETLIB_LAPACK_DIR)/TIMING
- endif
- 
- 
- lapack-test :
- 	(cd $(NETLIB_LAPACK_DIR)/TESTING && rm -f x* *.out)
--	make -j 1 -C $(NETLIB_LAPACK_DIR)/TESTING xeigtstc  xeigtstd  xeigtsts  xeigtstz  xlintstc  xlintstd  xlintstds  xlintstrfd  xlintstrfz  xlintsts  xlintstz  xlintstzc xlintstrfs xlintstrfc
-+	$(MAKE) -j 1 -C $(NETLIB_LAPACK_DIR)/TESTING xeigtstc  xeigtstd  xeigtsts  xeigtstz  xlintstc  xlintstd  xlintstds  xlintstrfd  xlintstrfz  xlintsts  xlintstz  xlintstzc xlintstrfs xlintstrfc
- ifneq ($(CROSS), 1)
- 	( cd $(NETLIB_LAPACK_DIR)/INSTALL; ./testlsame; ./testslamch; ./testdlamch; \
-         ./testsecond; ./testdsecnd; ./testieee; ./testversion )
-@@ -299,7 +299,7 @@ lapack-runtest:
- 
- blas-test:
- 	(cd $(NETLIB_LAPACK_DIR)/BLAS && rm -f x* *.out)
--	make -j 1 -C $(NETLIB_LAPACK_DIR) blas_testing
-+	$(MAKE) -j 1 -C $(NETLIB_LAPACK_DIR) blas_testing
- 	(cd $(NETLIB_LAPACK_DIR)/BLAS && cat *.out)
- 
```

Comment:
Yes, I realized that you have deleted openblas-0.2.19.  Could you please keep the openblas-0.2.19 for a while?